src/shims/fs.rs

```diff
@@ -257,6 +263,214 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         this.try_unwrap_io_result(result)
     }
 
+    fn statx(
+        &mut self,
+        dirfd_op: OpTy<'tcx, Tag>,
```

Comment:
why do these args have `_op` suffixes?

Reply:
is how I've written the arguments for almost all of the fs shims.

Reply:
Maybe I just noticed because `statx` has So. Many. Arguments.

Idk, no strong opinion on whether to keep or change, just seemed odd to me

Reply:
Oh yes, I try to preserve the names the arguments have in `man` so it is easier for someone else to check what's going on.